charts/opensearch-operator/values.yaml

```diff
@@ -3,11 +3,32 @@ tolerations: []
 manager:
   resources:
     limits:
-      cpu: 2
-      memory: 1Gi
+      cpu: 100m
+      memory: 300Mi
     requests:
-      cpu: 2
-      memory: 1Gi
+      cpu: 20m
```

Comment:
Just 20m CPU is very low. If users need to squeeze out every bit they can lower it themselves but I think the default should be higher.
My suggestion for CPU: requests: 100m, limit 200m.
Maybe add a comment that 20m is a minimum.